docstring_parser/parser/__init__.py

```diff
@@ -0,0 +1,41 @@
+"""Docstring parsing."""
+
+from . import rest
+from . import google
+from .common import ParseError, Docstring
+
+_styles = {"rest": rest.parse, "google": google.parse}
+
+
+def _parse_score(docstring: Docstring) -> int:
+    """
+    Produce a score for the parsing.
+
+    :param Docstring docstring: parsed docstring representation
+    :returns int: parse score, higher is better
+    """
+
+    return len(docstring.meta)
+
+
+def parse(text: str, style: str = 'auto') -> Docstring:
```

Comment:
Please make `style` into an `enum` rather than `str`

Reply:
I don't usually use `enum.Enum`. Did you mean something like this?
```python
class Style(enum.Enum):
    REST = rest.parse
    GOOGLE = google.parse
    AUTO = None


def parse(text: str, style: Style = Style.AUTO) -> Docstring:
    if style == Style.AUTO:
        return style.value(text)
    ...
```

Reply:
Yes, except the enum members can be in lowercase.

Reply:
The [documentation for `enum`](https://docs.python.org/3/library/enum.html#enum-member-type
) recommends upper-case enumeration members to prevent confusion with member methods

Reply:
I never liked that recommendation, but if you prefer it that way, it's okay :)

Reply:
Lower-case it is